MemberExpectations.md

```diff
@@ -0,0 +1,58 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the code of conduct but that supports
+the vision, mission and operating principles of the Node.js project.
+
+While members have the right and responsibility to exercise independent
+judgment and to express dissenting opinions during discussions within
+the organization, members also have the obligation outside the
+organization (twitter, reddit, etc.) to respect and support decisions
+and values of the majority, even when a member dissented from
```

Comment:
This `respect and support decisions of the majority view` is problematic. Perhaps `respect and support decisions made within the project's documented processes` or something similar.